src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,38 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+type Props = {
+  selectedTodo: Todo,
+  onCloseModal: (value: null) => void,
+};
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo, onCloseModal }) => {
+  const [user, setUser] = useState<User>();
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const getUserTodo = async () => {
+      try {
+        const userData = await getUser(selectedTodo.userId);
+
+        setUser(userData);
+        setIsLoading(false);
+      } catch (error) {
+        setIsLoading(false);
+      }
+    };
+
+    getUserTodo();
+  }, [selectedTodo]);
```

Comment:
i doubt `selectedTodo` can change with modal shown
```suggestion
  }, []);
```